Monika After Story/game/script-topics.rpy

```diff
@@ -13165,3 +13166,30 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_songwriting", 
+            category=["music"],
+            prompt="Songwriting", 
+            random=True, 
+            pool=True
+        )
+    )
+
+label monika_songwriting:
+    m 3eub "Hey [player], have you ever written a song?"
+    m 3eua "I've tried myself, and it's pretty fun. It takes a bit of time to do, but the end product is definitely worth it!"
+    m 4eub "You know, I used to write songs all the time in my sophomore years. I used to be pretty good!"
+    m 3eksdla "But, I guess it's been a while, huh?"
+    m 3eub "Anyway..."
+    m 1esb "You know, I think it'd be pretty cool if you ever dedicated a song to someone."
+    m 1hub "Especially if that person is me, ehehe!"
```

Comment:
Another thing to note, usually a `b` mouth is associated more with `Ahaha!` instead of `Ehehe!` since if mouthed out yourself, you'd notice that the `ahaha` will use an open mouth, while `ehehe` uses a closed one.

Reply:
Ah, I see! I didn't realize that! Would you recommend me instead using an "a" mouth, then?

Reply:
In this case, personally I'd use `Ahaha!`, could have it on it's own line and use a more teasing expression on this line if you wanted